provider/resources.go

```diff
@@ -128,67 +130,145 @@ func Provider() tfbridge.ProviderInfo {
+			"dynatrace_log_events":                     {Tok: dynatraceResource(mainMod, "LogEvents")},
+			"dynatrace_log_grail":                      {Tok: dynatraceResource(mainMod, "LogGrail")},
+			"dynatrace_log_metrics":                    {Tok: dynatraceResource(mainMod, "LogMetrics")},
+			"dynatrace_log_oneagent":                   {Tok: dynatraceResource(mainMod, "LogOneagent")},
+			"dynatrace_log_processing":                 {Tok: dynatraceResource(mainMod, "LogProcessing")},
+
+			"dynatrace_maintenance":              {Tok: dynatraceResource(mainMod, "Maintenance")},
+			"dynatrace_maintenance_window":       {Tok: dynatraceResource(mainMod, "MaintenanceWindow")},
+			"dynatrace_management_zone":          {Tok: dynatraceResource(mainMod, "ManagementZone")},
+			"dynatrace_mobile_application":       {Tok: dynatraceResource(mainMod, "MobileApplication")},
+			"dynatrace_network_zones":            {Tok: dynatraceResource(mainMod, "NetworkZones")},
+			"dynatrace_notification":             {Tok: dynatraceResource(mainMod, "Notification")},
+			"dynatrace_ops_genie_notification":   {Tok: dynatraceResource(mainMod, "OpsGenieNotification")},
+			"dynatrace_pager_duty_notification":  {Tok: dynatraceResource(mainMod, "PagerDutyNotification")},
+			"dynatrace_processgroup_naming":      {Tok: dynatraceResource(mainMod, "ProcessgroupNaming")},
+			"dynatrace_queue_manager":            {Tok: dynatraceResource(mainMod, "QueueManager")},
+			"dynatrace_queue_sharing_groups":     {Tok: dynatraceResource(mainMod, "QueueSharingGroups")},
+			"dynatrace_request_attribute":        {Tok: dynatraceResource(mainMod, "RequestAttribute")},
+			"dynatrace_request_naming":           {Tok: dynatraceResource(mainMod, "RequestNaming")},
+			"dynatrace_request_namings":          {Tok: dynatraceResource(mainMod, "RequestNamings")},
+			"dynatrace_resource_attributes":      {Tok: dynatraceResource(mainMod, "ResourceAttributes")},
+			"dynatrace_service_anomalies":        {Tok: dynatraceResource(mainMod, "ServiceAnomalies")},
+			"dynatrace_service_naming":           {Tok: dynatraceResource(mainMod, "ServiceNaming")},
+			"dynatrace_service_now_notification": {Tok: dynatraceResource(mainMod, "ServiceNowNotification")},
+			"dynatrace_slack_notification":       {Tok: dynatraceResource(mainMod, "SlackNotification")},
+			"dynatrace_slo":                      {Tok: dynatraceResource(mainMod, "Slo")},
+			"dynatrace_span_attribute":           {Tok: dynatraceResource(mainMod, "SpanAttributed")},
+			"dynatrace_span_capture_rule":        {Tok: dynatraceResource(mainMod, "SpanCaptureRule")},
+			"dynatrace_span_context_propagation": {Tok: dynatraceResource(mainMod, "SpanContextPropagation")},
+			"dynatrace_span_entry_point":         {Tok: dynatraceResource(mainMod, "SpanEntryPoint")},
+			"dynatrace_trello_notification":      {Tok: dynatraceResource(mainMod, "TrrelloNotification")},
```

Comment:
```suggestion
			"dynatrace_trello_notification":      {Tok: dynatraceResource(mainMod, "TrelloNotification")},
```

Typo in the original mapping.